src/Curve.elm

```diff
@@ -311,13 +315,14 @@ basisClosed : List (Vec2 Float) -> SubPath
 basisClosed points =
     let
         --| The ideal case where there are at least 3 points
-        commonCase close points =
+        commonCase : List (Vec3 (Vec2 Float)) -> (Vec2 Float -> Vec2 Float -> List (Vec3 (Vec2 Float))) -> List (Vec2 Float) -> List (Vec3 (Vec2 Float))
+        commonCase acc close points =
             case points of
                 p0 :: p1 :: [] ->
-                    close p0 p1
+                    List.reverse (List.reverse (close p0 p1) ++ acc)
```

Comment:
This looks like this could be simplified, making it a bit faster. 

I think this is a correct derivation
```
List.reverse (a ++ b) 
    = List.reverse b ++ List.reverse a
    -- a = List.reverse (close p0 p1), List.reverse << List.reverse = identity
    = List.reverse acc ++ close p0 p1
```

Reply:
That's indeed equivalent - but I (wrongly?) assumed that `acc` would usually be much longer than `close p0 p1`, we'd only pay the price of walking along `acc` once.